src/Components/Gene/GeneArtworksContent.tsx

```diff
@@ -30,7 +31,7 @@ export class GeneArtworksContent extends React.Component<Props, null> {
         const newLength = this.props.filtered_artworks.artworks.edges.length
         const newHasMore = this.props.filtered_artworks.artworks.pageInfo.hasNextPage
         if (newLength - origLength < PageSize && newHasMore) {
-          console.error("PERCY")
+          console.error(`Stopping gene artwork pagination early for gene with id: ${this.props.geneID}`)
```

Comment:
This error message could maybe better indicate what the problem is, rather than what that ends up doing in the UI. i.e. report that the count being returned doesn’t match the actual number of records we got back.